### PR TITLE
Fix pipeline fails when trying to trigger the jakartaee-documentation pipeline

### LINKED ISSUE #25: jakartaee-documentation-ui pipeline fails when trying to trigger the jakartaee-documentation pipeline
This is happening because the repo name has changed. We may need a new access token too, which may require help from the Eclipse folks since we don't have admin access to the jakartaee-documentation repo. 

See: https://github.com/jakartaee/jakartaee-documentation-ui/actions/runs/8319926729. This is the pipeline step: https://github.com/jakartaee/jakartaee-documentation-ui/blob/main/.github/workflows/build.yml#L62.

This should be easy for someone who has worked with Github Actions or Bitbucket Pipelines or something similar.

### PR DESCRIPTION
**Pull request by Ndacyayisenga-droid**
cc @kito99 

fixes https://github.com/jakartaee/jakartaee-documentation-ui/issues/25

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,72 +1,72 @@
...
 
-    - name: If pushed to main or tagged, publish UI bundle
-      if: github.event_name == 'push'
-      uses: softprops/action-gh-release@v1
-      with:
-        name: Latest UI bundle
-        files: build/ui-bundle.zip
-        tag_name: ${{ startsWith(github.ref, 'refs/tags/') && github.ref_name || 'latest' }}
+      - name: If pushed to main, set 'latest' tag on current build
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
```
Comment (kito99) [inline .github/workflows/build.yml]: @Ndacyayisenga-droid why did you change to `github-tag-action` from `action-gh-release`? That action was working just fine, and `githu-tag-action` doesn't do the same thing; it creates a tag, but doesn't create a release artifact (the ZIP file).

**Review by kito99 — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -1,72 +1,72 @@
...
+          files: build/ui-bundle.zip
+          tag_name: ${{ startsWith(github.ref, 'refs/tags/') && github.ref_name || 'latest' }}
+
+      - name: If published, trigger redeployment of playbook
+        if: github.event_name == 'push'
+        uses: convictional/trigger-workflow-and-wait@v1.6.5
+        with:
+          github_token: ${{ secrets.WORKFLOW_ACCESS_TOKEN }}
+          owner: ${{ github.repository_owner }}
+          repo: jakartaee-documentation
```
Comment (kito99) [inline .github/workflows/build.yml]: Ah! It's the name change. That makes sense...

File: .github/workflows/build.yml (hunk 1 above)
Comment (Ndacyayisenga-droid) [reply .github/workflows/build.yml]: Oooops some vs code extension refactored the yml file. Got it fixed

**Review by kito99 — APPROVED** [review]
